src/annotator/test/sidebar-test.js

```diff
@@ -559,10 +559,53 @@ describe('Sidebar', () => {
   });
 
   it('shows toolbar controls when using the default theme', () => {
-    createSidebar({});
+    createSidebar();
     assert.equal(fakeToolbar.useMinimalControls, false);
   });
 
+  it('hides the sidebar if window width is resized < MIN_RESIZE', () => {
+    const sidebar = createSidebar();
+    sidebar.show();
+    assert.isNotEmpty(sidebar.frame.style.marginLeft);
+
+    window.innerWidth = MIN_RESIZE - 1;
+    window.dispatchEvent(new Event('resize'));
+    assert.isEmpty(sidebar.frame.style.marginLeft);
+  });
+
+  it('updates marginLeft CSS style of the sidebar if sidebar is open and window width is resized > MIN_RESIZE', () => {
```

Comment:
I'm having trouble understanding this test description (and what's in the test). It might help to make the test description describe what the expected outcome is, not the implementation of that outcome, if that makes sense?

Reply:
Yes, I get your point.

Ideally, it should be something like this:

```
    it('updates marginLeft of the sidebar if window is resized', () => {
      const sidebar = createSidebar();
      sidebar.show();

      // Make the window very small
      window.innerWidth = MIN_RESIZE;
      window.dispatchEvent(new Event('resize'));
      const marginLeftInitial = sidebar.frame.style.marginLeft;

      // Make the window very large
      window.innerWidth = MIN_RESIZE * 10;
      window.dispatchEvent(new Event('resize'));
      const marginLeftFinal = sidebar.frame.style.marginLeft;

      // Compare marginLeft values
      assert.notEqual(marginLeftInitial, marginLeftFinal);
    });
```

However, it is a little bit more complicated because the media query styles are not applied when for example the screen become very small.